docs/src/main/sphinx/release/release-351.md

```diff
@@ -0,0 +1,69 @@
+# Release 351 (3 Jan 2021)
+
+## General Changes
+
+* Rename client protocol headers to start with `X-Trino-`.
+  Legacy clients can be supported by setting the configuration property
+  `protocol.v1.alternate-header-name` to `Presto`. This configuration
+  property is deprecated and will be removed in a future release.
+
+## JMX MBean Naming Changes
+
+* Rename base domain name for server MBeans to `trino`. The name can
+  be changed using the configuration property `jmx.base-name`.
+* Rename base domain name for the Elasticsearch, Hive, Iceberg, Raptor,
+  and Thrift connectors to `trino.plugin`. The name can be changed
+  using the catalog configuration property `jmx.base-name`.
+
+## Server RPM Changes
+
+* Rename installation directories from `presto` to `trino`.
+
+## Docker Image Changes
+
+* Publish image as [`trinodb/trino`](https://hub.docker.com/r/trinodb/trino).
+* Change base image to `azul/zulu-openjdk-centos`.
+* Change configuration directories to `/usr/lib/trino/etc` and `/etc/trino`.
```

Comment:
As we are breaking things anyway, this is a good occasion to clean things up and keep only one dir as supported.